ui/StatusQ/src/StatusQ/Components/StatusRoundedComponent.qml

```diff
@@ -0,0 +1,36 @@
+import QtQuick 2.13
+import QtGraphicalEffects 1.0
+import StatusQ.Core 0.1
+import StatusQ.Core.Theme 0.1
+
+Rectangle {
```

Comment:
Suggestions: `Control` as a base class, `Rectangle` as  the `background` and `Loader` as the `contentItem` so we can take advantage of control functionality!

Reply:
hmm interesting points... @noeliaSD what features of `Control` do you think this component would benefit from?

Reply:
Sizing issues, focus control, hover property, proper padding spacing, etc. Pls inherit from Control; doing a component with a `Rectangle` also leaks its API to the consumers; when you later decide to change your mind, properties like `color` or `radius` are suddenly no longer available, etc...

Reply:
One more thing: I did actually want all those properties from `Rectangle` to be available in all components inheriting from `StatusRoundComponent` in order not to break the API of `StatusRoundedImage`. Properties like `border`, `radius`, `color` are used in many instances of that one.

Reply:
Right... I see. Then maybe don't make it public?

Reply:
I will not be able to add more extra things in the list that @caybro mentioned to convence you! :blush:  

As  @IvanBelyakoff posted `Control is the base type of user interface controls` and just one simple example of benefit is the sizing management commented above. Yes, with `Rectangle` you also can manage it, but why we need to "make it" for example via margins, if we already have the proper porperty for that if it is a `Control`?

It is more flexible and you will avoid to do some "handmade" trickes in some point. 

It gives you a more extensive api just for free, and I don't see disadvantatges on that. If in some point we need to take care of event handling, you will just be able to use it, if now it is not the case / need, just ignore that.

We are building here a reusable component for `StatusQ` so, why not having the more extensive api possible (efforless :wink:)?

However, it was just a suggestion and feel completely free to decide whatever you thing is better or you feel more comfortable! :blush: